library/x509_crt.c

```diff
@@ -2938,6 +2938,7 @@ static int x509_crt_find_parent_in(
                         mbedtls_x509_crt_restart_ctx *rs_ctx )
 {
     int ret;
+    volatile int ret_fi;
```

Comment:
Should this be initialised to a failed return value?

Reply:
Good point. I'll fix that.